core/services/keeper/validate_test.go

```diff
@@ -115,49 +75,112 @@ encode_check_upkeep_tx -> check_upkeep_tx -> decode_check_upkeep_tx -> encode_pe
 			},
 			wantErr: false,
 		},
+
 		{
-			name: "invalid job spec",
+			name: "invalid job spec because of type",
 			args: args{
 				tomlString: `
-type            = "keeper"
-schemaVersion   = 3
-name            = "example keeper spec"
-contractAddress = "0x9E40733cC9df84636505f4e6Db28DCa0dC5D1bba"
-fromAddress     = "0xa8037A20989AFcBC51798de9762b351D63ff462e"
-evmChainID      = 4
-externalJobID   =  "123e4567-e89b-12d3-a456-426655440002"
+						type            = "vrf"
+						name            = "invalid keeper spec example 1"
```

Comment:
formatting

Reply:
Could be that I'm getting lost with all these formatting issues haha but isn't it correct here?

Reply:
sorry, you're right